content/sql/concepts/math-functions/terms/atan/atan.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
You still need to update the metadata to be properly formatted. The rules for the metadata can be found [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata).